### PR TITLE
Fix type hints for older Python 3.5.

### PR DESCRIPTION
**Pull request by ueshin**
Fixes type hints for older Python 3.5.

### CODE REVIEW

**Review by xinrong-databricks — APPROVED** [review]

File: databricks/koalas/__init__.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def assert_pyspark_version():
     logging.warning(
         "'PYARROW_IGNORE_TIMEZONE' environment variable was not set. It is required to "
         "set this environment variable to '1' in both driver and executor sides if you use "
-        "pyarrow>=2.0.0."
+        "pyarrow>=2.0.0. "
```
Comment (xinrong-databricks) [inline databricks/koalas/__init__.py:82]: Just wanted to make sure this is intentional.

Comment (ueshin) [reply databricks/koalas/__init__.py:82]: Yes, otherwise the message doesn't have an space between sentences.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by ueshin** [comment]
Thanks! I'd merge this now.

**Comment by itholic** [comment]
Nice!

**Comment by xinrong-databricks** [comment]
Great, thank you!

**Pull request merged**